build-tools/pnpm-lock.yaml

```diff
@@ -1227,9 +1227,6 @@ packages:
     engines: {node: '>=18'}
     peerDependencies:
       '@types/node': '>=18'
-    peerDependenciesMeta:
-      '@types/node':
-        optional: true
```

Comment:
There are many instances of this change; do we know why they happened? With no changes to the `@inquirer/checkbox` dependency itself, I don't get why this section is disappearing